roulette.js

```diff
@@ -5,12 +5,27 @@ class RouletteBetting {
     }
 
     placeBet(name, numbers, betAmount) {
-        numbers = numbers.split(',').map(num => parseInt(num, 10));
-        betAmount = parseInt(betAmount, 10);
-        this.bets.push({ name, numbers, betAmount });
+        if (name === 'red' || name === 'black' || name === 'even' || name === 'odd') {
+            this.placeSpecialBet(name, betAmount);
+        } else {
+            numbers = numbers.split(',').map(num => parseInt(num, 10));
+            betAmount = parseInt(betAmount, 10);
+            this.bets.push({ name, numbers, betAmount });
+        }
         this.updateCurrentBetsDisplay();
     }
 
+    placeSpecialBet(name, betAmount) {
+        const specialBets = ['red', 'black', 'even', 'odd'];

```

Comment:
Instead of special bets, call them outside beds and include `1-12`, `13-24` and `25-36` as well as red, black, even and odd.